.github/workflows/pythonpackage.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       # Label used to access the service container
       vault:
         # Docker Hub image
-        image: vault
+        image: hashicorp/vault:latest
```

Comment:
We should pin this image to a specific version, not latest.

Reply:
good point